submissions/yusiratArogundade/htmlForm/signUp.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Sign up page</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="../css/style.css">
+	<link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
+</head>
+<body>
+	<div class="nav-wrapper">
+		<div class="nav-logo">
+				<img src="../HTML/images/logo.png">
+		</div>
+		<div>
+			<nav>
+				<ul>
+					<li><a href="../HTML/index.html">Product</a></li>
+					<li><a href="../htmlForm/contactUs.html">Contact</a></li>
+					<li><a href="../htmlForm/signUp.html">Sign Up</a></li>
+				</ul>
+			</nav>
+		</div>	
+	</div>
+	<div class="content-wrapper">
+		<div class="reg-info">
+			<h1>Register your account</h1>
+			<p>Registering an account gives you <br>
+				access to sweet promo and events <br>
+				available to only registered <br>
+				customers
+			</p>
+			<p>Already got an account? <strong><a href="#">Sign in</a></strong></p>
+		</div>
+		<div class="sign-up-form">
+			<form>
+					<fieldset id="form-rec">
+						<label for="first_name">First Name</label><br/>
+						<input type="text" name="first_name" id="first_name"><br/>
+						<br/>
+						<label for="last_name">Last Name</label><br/>
+						<input type="text" name="last_name" id="last_name"><br/>
+						<br/>
+						<label for="others_name">Others Name</label><br/>
+						<input type="text" name="others_name" id="others_name"><br/>
+						<br/>
+						<label for="phone_no">Phone Number</label><br/>
+						<input type="Number" name="phone_no" id="phone_no"><br/>
```

Comment:
It think you should make this input type should be `text` in this case, not `number`. @yusrah247